src/styles/blocks/main.scss

```diff
@@ -0,0 +1,9 @@
+.catalog {
+  --column-count: 4;
```

Comment:
According to the [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), custom properties should be placed in a separate file `variables.scss`. Consider moving `--column-count` to `variables.scss` for better maintainability and reusability.